src/OAuthClient/PatronClient.php

```diff
@@ -40,4 +40,29 @@ public static function getPatronById($patronId = '')
             return null;
         }
     }
+
+    public static function notifyPatron(Patron $patron, int $holdId)
+    {
+        $url = Config::get('API_PATRON_URL') . '/' . $patron->getId() . '/notify';
+        $payload = array('holdRequestServiceId' => $holdId, 'type' => 'hold-success');
+        APILogger::addInfo("Calling PatronServices/notify: $url " . json_encode($payload));
+
+        $response = ClientHelper::postResponse($url, $payload, __FUNCTION__);
+        $statusCode = $response->getStatusCode();
+
+        $response = json_decode((string)$response->getBody(), true);
+
+        APILogger::addInfo('Got response from patronservices notify endpoint ' . json_encode($response));
+
+        // Check statusCode range
+        if ($statusCode === 200) {
+          return true;
+        } else {
+            APILogger::addError(
+                'Failed',
+                array('Failed to call patronservices notify endpoint')
```

Comment:
this might not be the most accurate error message. we return a bunch of 4xx errors, so its not that the endpoint was not called, but that there was some error in that workflow.

Reply:
Agreed. Improved that log line [here](https://github.com/NYPL/hold-request-result-consumer/pull/60/files#diff-fa1aee2ac94bd292c8f94eaf68ec6f3b1a6bb6ec23c7b8e6005392fb22078c2cR64)